Examples/Algorithms/Vertexing/src/AdaptiveMultiVertexFinderAlgorithm.cpp

```diff
@@ -135,8 +135,7 @@ ActsExamples::AdaptiveMultiVertexFinderAlgorithm::executeAfterSeederChoice(
     // maximum chi2 that a track can have to be associated with a vertex).
     finderConfig.tracksMaxSignificance = 7.5;
     // Check if vertices are merged in space and time
-    // TODO rename do3dSplitting -> doFullSplitting
-    finderConfig.do3dSplitting = true;
+    finderConfig.doFullSplitting = true;
```

Comment:
@pbutti argued that this should be harmonized between time and no time. can we drop this? like I did in https://github.com/acts-project/acts/pull/2987

Reply:
we can do this in another PR